x11/turbovnc-viewer/Portfile

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           java 1.0
+
+github.setup        TurboVNC turbovnc 2.2.6
+
+name                turbovnc-viewer
+platforms           darwin
+license             GPL-2
+maintainers         {gmail.com:audvare @Tatsh}
+categories          x11 net
+description         TurboVNC VNC viewer.
+long_description    ${description}
+
+checksums           rmd160  de684a6e9c2f2e1ef1f05b98229fd40269d17a7c \
+                    sha256  71a89600d1e4ffe67aa65e5ddc6b21b1659b9105892c07074e99181218dc33e8 \
+                    size    9023665
+
+depends_lib         port:libjpeg-turbo
+
+java.version        1.8+
+
+# This must be called to set ${java.home}
+java_set_env
```

Comment:
Unfortunately this call will cause the port to seem different on every `portindex`. I am not sure how to resolve this.

Reply:
I put this in `pre-configure` along with `configure.cmd` and it seems to work.